test/config/utility.cc

```diff
@@ -396,10 +397,17 @@ void ConfigHelper::initializeTls(envoy::api::v2::auth::CommonTlsContext& common_
   validation_context->add_verify_certificate_hash(TEST_CLIENT_CERT_HASH);
 
   auto* tls_certificate = common_tls_context.add_tls_certificates();
-  tls_certificate->mutable_certificate_chain()->set_filename(
-      TestEnvironment::runfilesPath("/test/config/integration/certs/servercert.pem"));
-  tls_certificate->mutable_private_key()->set_filename(
-      TestEnvironment::runfilesPath("/test/config/integration/certs/serverkey.pem"));
+  if (ecdsa_cert) {
```

Comment:
Does this have to be either or? Once rest of your changes land, we're going to have ability to set both.

Perhaps pass `rsa_cert` and `ecdsa_cert`, and guard that it's either or in this commit?

Reply:
Yeah, eventually it will be both, but we will also want additional flexibilty with multi-cert, e.g. to have multiple ECDSA certs in testing to verify we only pick the first. So, I think we can leave it like this for this PR and then add this in later.